docusaurus.config.js

```diff
@@ -25,6 +25,53 @@ module.exports = {
   favicon: 'img/meta/favicon-96x96.png',
   organizationName: 'ionic-team',
   projectName: 'ionic-docs',
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      {
```

Comment:
The command looks really big when compared with the "K"

| production | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/abe17c9a-f254-4583-ae72-8d04811389a6) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/bad25eb3-6662-455b-b0db-d5060219102d) |

Reply:
Hmm this still doesn't look quite right: 
![image](https://github.com/ionic-team/ionic-docs/assets/2721089/5d28362a-576a-4bc1-870a-6c118fb77ca2)

The command logo and the letter seem to be different sizes/misaligned.

Reply:
does it still look weird? I made changes last Friday: https://github.com/ionic-team/ionic-docs/pull/3014/commits/72a0ccf06021475f1b43a8f764666e92b1dab9c4

Reply:
It looks a lot better now! 
![image](https://github.com/ionic-team/ionic-docs/assets/2721089/f1aeb50e-ce44-4dbd-8d6e-5a1d8625e391)